lib/reporters/base.js

```diff
@@ -27,7 +27,13 @@ var isatty = tty.isatty(1) && tty.isatty(2);
 /**
  * Enable coloring by default, except in the browser interface.
  */
-
+if (process.env.MOCHA_COLORS) {
+  utils.deprecate(
+    '"MOCHA_COLORS" is deprecated and will be removed from a future version of Mocha. Use "FORCE_COLOR" instead.'
+  );
+  delete process.env.MOCHA_COLORS;
+  process.env.FORCE_COLOR = 1;
```

Comment:
Environment variables should _always_ be strings...

Reply:
But this is a rather [heavy-handed](https://www.youtube.com/watch?v=QGc-iPc-9dE) approach...